docs/samples/Microsoft.ML.Samples/Microsoft.ML.Samples.csproj

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <OutputType>Exe</OutputType>
-    <SignAssembly>false</SignAssembly>
-    <PublicSign>false</PublicSign>
```

Comment:
I think this properties were added by @artidoro for a reason.

Reply:
Right. This ensures that we can never make the mistake of adding this as a friend assembly... please don't remove...

Reply:
Can you also please add a comment explaining this for posterior knowledge.